### ISSUE TITLE
stop game

### ISSUE THREAD

**Issue by mavthedev** [leopard-js/leopard#112]
is there a way for me to stop the game and will there ever be? such as `project.detach()` or `project.stop()`
i would like this very much

**Comment by towerofnix** [comment]
We're tracking this feature in #124. It's in planned, just not implemented yet!
